runtime/compiler/optimizer/SequentialStoreSimplifier.cpp

```diff
@@ -1762,33 +1758,17 @@ static TR::TreeTop* generateArraysetFromSequentialStores(TR::Compilation* comp,
                constValueNode = TR::Node::bconst(istoreNode,(int8_t)arrayset.getConstant());
                break;
                }
-            case 2:
```

Comment:
Again I don't think we should be deleting this case I think it needs updating...

Reply:
Should have changed `cstorei` to `sstorei` instead of not handling the whole 2 bytes case. Going to fix this in the new force push.